### DISCUSSION TITLE
How to re-registe application commands

### DISCUSSION THREAD

**Discussion by lol219** [Rapptz/discord.py#8074] [Q&A] [answered]
Hello Guys,
I have a question, how can I re-register the application commands.
Thanks!

**Comment by GoogolGenius** [comment, marked as answer]
Just call the `~.CommandTree.sync()` method again ?

**Reply by lol219** [reply]
In async setup_hook or

**Reply by GoogolGenius** [reply]
Well you *can*, but it's unnecessary. You only need to send a request to the API when you add a new command, update the options, names, etc. __about the command__. Not what the command actually does aka in the function in most cases. 

You can create a prefix command or some simple listener for even your bot's prefix to sync the command tree. 

You can sync right now in setup_hook for quick purposes but consider the above too.
